src/pe_reports/stakeholder/views.py

```diff
@@ -194,14 +189,14 @@ def setCustRootDomain(customer, rootdomain, orgUUID):
             return True
 
     except (Exception, psycopg2.DatabaseError) as err:
-        logging.error("There was a problem logging into the psycopg database %s", err)
+        LOGGER.error("There was a problem logger into the psycopg database %s", err)
```

Comment:
Overzealous search/replace correction:
```suggestion
        LOGGER.error("There was a problem logging into the psycopg database %s", err)
```